RealmSwift/Combine.swift

```diff
@@ -339,10 +337,20 @@ extension RealmCollection where Self: RealmSubscribable {
         RealmPublishers.Value(self)
     }
 
+    /// A publisher that emits the collection each time the collection changes on the given property keyPaths.
+    @discardableResult public func collectionPublisher(keyPaths: [String]?) -> RealmPublishers.Value<Self> {
```

Comment:
I don't think `@discardableResult` makes sense here? When would you want to call this and not use the return value?

Reply:
You're right. I think I added it for something strange. I'll remove.